### PR TITLE
report-service: mongodb init

### PR DESCRIPTION
**Pull request by bheptinh**

### CODE REVIEW

File: report-service/index.ts [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+import mongoose from 'mongoose';
+import router from './routes';
 
-import router from './routes'
+const HOST = 'localhost';
+const PORT = '16717';
+const DATABASE = 'report-service';
+
+mongoose
+    .connect(`mongodb://${HOST}:${PORT}/${DATABASE}`, { useNewUrlParser: true })
```
Comment (tmorozov) [inline report-service/index.ts]: instead of 3 variables - we could use 1 (connection string) 
and we also could make it part of environment

Comment (fylhsq123) [reply report-service/index.ts]: Good idea. I think we can use 'config' module here.

Comment (bheptinh) [reply report-service/index.ts]: I believe we'll use docker here and probably we will get port, database, etc. from env, that's why 3 variables here

Comment (fylhsq123) [reply report-service/index.ts]: Then you should use them like environment variables.
process.env.HOST
process.env.PORT
process.env.DATABASE

But nevertheless you can use config module, where we can define some configurations and override them with environment variables later.
Something like they do here:
https://github.com/lorenwest/node-config/wiki/Environment-Variables#custom-environment-variables

Comment (bheptinh) [reply report-service/index.ts]: moved variables to `.env` file, added new module `dotenv`

**Pull request merged**